### ISSUE TITLE
Request Body for POST /shell-descriptors

### ISSUE THREAD

**Issue by Yasuhito-Yoshida-3** [admin-shell-io/aasx-server#189]
How do I set the Request Body for POST /shell-descriptors?
I would like a concrete example. I don't know the relationship with the existing Example_AAS_ServoDCMotor_21.aasx file.
Does anyone know?
I don't understand as there is no detailed description in IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf.
I would like to know more about AssetAdministrationShellRegistryAPIApi.

**Comment by aorzelskiGH** [comment]
Please take a look on the demo servers: https://v3.admin-shell-io.com/ and https://v3registry.admin-shell-io.com .
Get the shell descriptors of the registry by https://v3registry.admin-shell-io.com/shell-descriptors .
Look on the API at /swagger of each server.
Please do not POST to the public demo registry, but set up your own one.
I have put 2 AASX examples into /examples folder: 1 for a local registry and 1 for an empty registry.
For an empty registry server just put the empty one as only AASX.
For a repositiory server put first the local AASX into and you see a local registry.
As a final step you add another postRegistry to create automatic POSTs to your own registry server.

**Comment by Yasuhito-Yoshida-3** [comment]
Thank you for your answer.
The following was confirmed on the local server.
/lookup/shells
/shell-descriptors
/shell-descriptors/{aasIdentifier}

However, the following returns status code 204.
/shell-descriptors/{aasIdentifier}/submodel-descriptors
There is no internal JSON data.
When I tried the following, I got the same 204 status code.
https://v3.admin-shell-io.com/swagger/index.html

Is it not implemented yet? Or do I need a proper AASX sample?
